src/browser_action/vitals.js

```diff
@@ -224,53 +233,144 @@
+          const navEntry = metric.attribution.navigationEntry;
+          // Set the start time to the later of the actual start time or the activationStart (for prerender) or 0
+          const startTime = Math.max(navEntry.startTime, navEntry.activationStart) || 0;
+          // Add the performance marks for the Performance Panel
+          if (enableUserTiming) {
+              performance.measure(`[Web Vitals Extension] LCP.timeToFirstByte`, {
+              start: startTime,
+              duration: metric.attribution.timeToFirstByte,
+            });
+            performance.measure(`[Web Vitals Extension] LCP.resourceLoadDelay`, {
+              start: startTime + metric.attribution.timeToFirstByte,
+              duration: metric.attribution.resourceLoadDelay,
+            });
+            performance.measure(`[Web Vitals Extension] LCP.resourceLoadTime`, {
+              start: startTime + metric.attribution.timeToFirstByte + metric.attribution.resourceLoadDelay,
+              duration: metric.attribution.resourceLoadTime,
+            });
+            performance.measure(`[Web Vitals Extension] LCP.elmentRenderDelay`, {
+              duration: metric.attribution.elementRenderDelay,
+              end: metric.value
+            });
+          }
+          // Add a nice console output
+          if (enableConsoleTables) {
+            console.table(
+              [
+                {
+                  'LCP Breakdown': `LCP (${metric.attribution.lcpEntry.element.nodeName})`,
+                  'Time (ms)': metric.value.toFixed(2),
+                },
+                {
+                  'LCP Breakdown': 'timeToFirstByte',
+                  'Time (ms)': metric.attribution.timeToFirstByte.toFixed(2),
+                },
+                {
+                  'LCP Breakdown': 'resourceLoadDelay',
+                  'Time (ms)': metric.attribution.resourceLoadDelay.toFixed(2),
+                },
+                {
+                  'LCP Breakdown': 'resourceLoadTime',
+                  'Time (ms)': metric.attribution.resourceLoadTime.toFixed(2),
+                },
+                {
+                  'LCP Breakdown': 'elementRenderDelay',
+                  'Time (ms)': metric.attribution.elementRenderDelay.toFixed(2),
+                }
+              ]
+            )
+          }
+        }
         break;
       case "CLS":
-        // CLS has a startTime, but not a duration.
-        // Could visualize the time between rendering tasks (Commit-to-Commit).
-        // Skip for now.
+        if (enableConsoleTables) {
+          // Add a nice console output of all the shifts
+          let entries = [{'CLS Breakdown': 'CLS', 'Time (ms)': metric.value}]
+          metric.entries.map((entry) => {
+            entries.push({'CLS Breakdown': entry.entryType, 'Time (ms)': entry.value});
+          });
+          console.table(entries)
+        }
```

Comment:
I'd suggest reverting the CLS and INP tables. Don't think they're as useful as the LCP use case and not in scope for #112 anyway.

Reply:
I disagree. As per comment above I see these tables as a friendly option than the Performance Entry logging, that could be presented without those. Or could be presented with, but concentrated on in the table and only falling back to the, more difficult-to-read, performance entries if need even more detail. So, with that rational, it seems odd to me to only include one.

I also think the INP table in particular IS useful to have that broken down to it's parts. It is likely we'd be adding that anyway so why not go with it from now.

I agree it can make the console a little noisy, but it's easy to filter by typing the metric name to just show those metrics. The alternative is to have an option per metric but think that's a little overkill.

Reply:
Can we compromise on removing the CLS table? Not even sure what to do with this info. We really need the attribution info to take any action on it.

<img width="573" alt="image" src="https://user-images.githubusercontent.com/1120896/232571176-44321b06-a61f-4739-aa32-9654a5631801.png">